index.html

```diff
@@ -258,8 +258,11 @@ <h1 class="form-heading">Contact me</h1>
             </p>
 
             <form action="https://formspree.io/f/mpznvrdy" method="post">
+                <label for="" class="d-none">Fullname</label>
                 <input type="text" placeholder="Your name" id="fullname" name="fullname" minlength="1" maxlength="30" required>
+                <label for="" class="d-none">Email</label>
                 <input type="email" name="email" placeholder="Email Address" id="email" required>
+                <label for="" class="d-none">Text Area</label>
                 <textarea name="message" id="message" minlength="1"  maxlength="500" placeholder="Enter text here" cols="30" rows="10" required></textarea>
 
                 <button type="submit" class="form-btn">Get in touch</button>
```

Comment:
- Kindly fix the contrast on the following buttons as well


_CASE 1: In this case, text(html) on default, hover, and focus are failing, kindly make the colors a bit darker, or whiter as the case may be_

|html text |
| :---:   | 
|![htm](https://user-images.githubusercontent.com/99246359/207813229-04986b60-f465-4139-8152-0e193ac929d8.PNG)  |




_CASE 2: In this case, you should make the text of see project button (on hover) a bit whiter_ 🤗 
| Your see project button  |
| :---:   | 
|![sep](https://user-images.githubusercontent.com/99246359/207813449-08a2e8bc-c708-423a-852f-b1aec034cdeb.PNG)  |




_CASE 3: In this case, fix the text as i have shown you how above._
| Your Get in touch button  |
| :---:   | 
|![gtt](https://user-images.githubusercontent.com/99246359/207813657-5ec8f958-4a78-4d3a-875a-503f3cf772c3.PNG)  |





When you are done, you can carry out the test yourself by pasting the link of your deployed site on this 👉🏽 [aditus tool](https://www.aditus.io/button-contrast-checker/)